doc/source/conf.py

```diff
@@ -286,5 +291,14 @@
 
 # -- Linkcheck config --------------------------------------------------------
 
-linkcheck_ignore = [r"https://github.com/ansys/pymechanical/pkgs/container/.*"]
+# linkcheck_ignore = [r"https://github.com/ansys/pymechanical/pkgs/container/.*"]
```

Comment:
If you have modified it to ignore more links, remove the old implementation. Unless necessary, commented out code is not a good practice...
```suggestion
```